pkg/newlog/cmd/newlogd.go

```diff
@@ -661,6 +663,23 @@ func startTmpfile(dirname, filename string) *os.File {
 	if err != nil {
 		log.Fatal(err)
 	}
+	// make symbolic link for device temp logfiles
+	if !isApp {
+		if _, err = os.Stat(tmpSymlink); err == nil {
```

Comment:
Why Stat when you can just call Remove right away?

Reply:
since the tmpSymlink is renamed every time, so it won't be there. the only time we need to remove this is it reboots at the time of this changing. yes i can use remove() always and get error and check if the error is NotExist(), but this Stat() is probably less work.